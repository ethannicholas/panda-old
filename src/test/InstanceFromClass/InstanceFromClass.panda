class Test {
    var instanceField:Int

    method instanceMethod():Int {
        return instanceField
    }

    @class 
    method main() {
        Console.write(instanceField)
        Console.write(instanceMethod())
    }
}