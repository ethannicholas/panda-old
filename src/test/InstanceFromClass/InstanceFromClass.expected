error: InstanceFromClass.panda:10:23: instance field 'instanceField' cannot be accessed from a class context
error: InstanceFromClass.panda:11:23: instance method 'instanceMethod' cannot be accessed from a class context
2 errors
